### PR TITLE
LOC CHECKIN | vstest | 20171002

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

**Comment by codito** [comment]
@dotnet-bot test Windows_NT / Debug Build  please

File: src/vstest.console/Resources/xlf/Resources.fr.xlf [hunk 1]
```diff
@@ -1495,64 +1495,57 @@
...
-        <note></note>
+        <target state="translated">De plus, vous pouvez spécifier le chemin des adaptateurs de test à l'aide de la commande /TestAdapterPath. Exemple : /TestAdapterPath:&lt;chemin_adaptateurs_personnalisés&gt;.</target>
+        <note />
       </trans-unit>
       <trans-unit id="UseVsixExtensionsDeprecation">
         <source>/UseVsixExtensions is getting deprecated. Please use /TestAdapterPath instead.</source>
-        <target state="new">/UseVsixExtensions is getting deprecated. Please use /TestAdapterPath instead.</target>
-        <note></note>
-      </trans-unit>
-      <trans-unit id="DisableAutoFakesUsage">
```
Comment (Faizan2304) [inline src/vstest.console/Resources/xlf/Resources.fr.xlf]: Why did we delete it from xlf file?

**Comment by Faizan2304** [comment]
@dotnet-bot test Windows_NT / Debug Build please

File: src/Microsoft.TestPlatform.Client/Resources/xlf/Resources.de.xlf [hunk 2]
```diff
@@ -39,9 +39,9 @@
         <note from="bb-metadata">fuzzyMatch="15" wordcount="14" adjWordcount="11.9" curWordcount="11.9"</note>
       </trans-unit>
       <trans-unit id="TestSessionTimeoutMessage">
-        <source>Aborting test run: test run timeout of {0} milliseconds exceeded.</source>
-        <target state="new">Canceling test run as it crossed testSessionTimeout: {0} milliseconds.</target>
-        <note></note>
+        <source>Canceling test run: test run timeout of {0} milliseconds exceeded.</source>
```
Comment (Faizan2304) [inline src/Microsoft.TestPlatform.Client/Resources/xlf/Resources.de.xlf]: this should be `Canceling test run: test run timeout of {0} milliseconds exceeded`.

**Comment by Faizan2304** [comment]
@dotnet-bot test Windows_NT / Debug Build please

**Comment by Faizan2304** [comment]
@dotnet-bot test Windows_NT / Release Build please

**Pull request merged**